src/main/java/it/gov/pagopa/ranking/dto/PageRankingDTO.java

```diff
@@ -0,0 +1,49 @@
+package it.gov.pagopa.ranking.dto;
+
+import lombok.*;
+import org.springframework.data.domain.PageImpl;
+import org.springframework.data.domain.PageRequest;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.domain.Sort;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@EqualsAndHashCode(callSuper = true)
+@Getter
+@Setter
+public class PageRankingDTO<T> extends PageImpl<T> {
+    private static final String RANKING_FIELD = "ranking";
+
+    private String rankingStatus;
+    private LocalDateTime rankingPublishedTimeStamp;
+    private LocalDateTime rankingGeneratedTimeStamp;
+
```

Comment:
let's add also the fields containing the number of onboardingOk and onboardingKo